src/ast.js

```diff
@@ -334,12 +337,9 @@ function seq_for_testlist (c, n) {
     var seq = [];
     Sk.asserts.assert(n.type === SYM.testlist ||
         n.type === SYM.testlist_star_expr ||
-        n.type === SYM.listmaker ||
```

Comment:
Which version would of python did you get this from? Lets change the name to be snake case too.

Reply:
I do not know where any of that came from...

Reply:
we should port this from https://github.com/python/cpython/blob/3.7/Python/ast.c#L1186-L1213 can you add a comment to the source. it doesn't make sense to do it in this PR but we're probably missing some edge cases 😄

Reply:
I'm not sure what you're asking for.  This is correctly handling "testlist", I think.  It just also had a check for "listmaker", which doesn't exist anywhere.  It's not in https://github.com/python/cpython/blob/3.7/Python/ast.c anywhere, either.

I'm fine with adding a comment if there is something missing, but I'm not sure what that comment should be...

Reply:
I didn't mean the listmaker specifically I meant that we should make sure this whole function is aligned with the py3.7 version.